lib/connect-couchbase.js

```diff
@@ -127,14 +127,13 @@ module.exports = function(session){
                     console.log("Could not connect to couchbase with bucket: " + connectOptions.bucket);
                     self.emit('disconnect', err);
                 } else {
+                    this.client.connectionTimeout = connectOptions.connectionTimeout || 10000;
```

Comment:
For the latest SDK, update this to:

this.connectionTimeout = connectOptions.connectionTimeout || 10000;
this.operationTimeout = connectOptions.operationTimeout || 10000;

Reply:
This is strange... or maybe I am missing something painfully obvious...
According to the documentation, connectionTimeout  and operationTimeout are still members of the bucket class in SDK version 2.6.1
[Link](http://docs.couchbase.com/sdk-api/couchbase-node-client-2.6.1/Bucket.html#connectionTimeout__anchor).

And another thing, "this" keyword in this context, will refer to the "owner" of this function, which doesn't have any of those members.